dio/pubspec.yaml

```diff
@@ -0,0 +1,22 @@
+name: sentry_dio
+description: An integration which adds support for performance tracing for the Dio package.
+version: 6.3.0
+homepage: https://docs.sentry.io/platforms/dart/
+repository: https://github.com/getsentry/sentry-dart
+issue_tracker: https://github.com/getsentry/sentry-dart/issues
+
+environment:
+  sdk: '>=2.12.0 <3.0.0'
+
+
+dependencies:
+  dio: ^4.0.0
+  sentry: 
+    path: ../dart
```

Comment:
I had to change the pure dart sdk, so I need to use a path dependency. Obviously, this must be changed before the release.

Reply:
This prevents usage as git dependency, better define the version here and add the path to dependency_overrides.

Reply:
Done